src/components/product/OutOfStock/OutOfStock.tsx

```diff
@@ -91,33 +94,31 @@ function OutOfStock(props: OutOfStockProps) {
   }
 
   return (
-    <section data-store-out-of-stock data-testid={testId} aria-live="polite">
-      <Form data-out-of-stock-form onSubmit={handleSubmit}>
-        <p className="text__title-subsection">{title}</p>
-        <p data-store-out-of-stock-subtitle>
-          {notificationMsgIcon} {notificationMsg}
-        </p>
-        <div>
-          <InputText
-            id="out-of-stock-email"
-            value={email}
-            label="Email"
-            aria-label="Email"
-            onChange={(e) => setEmail(e.target.value)}
-          />
-          <Button
-            data-store-out-of-stock-button
-            type="submit"
-            disabled={disabled}
-            variant="primary"
-            icon={buttonIcon}
-            iconPosition="left"
-          >
-            {buttonText}
-          </Button>
-        </div>
-      </Form>
-    </section>
+    <UIOutOfStock aria-live="polite" onSubmit={handleSubmit}>
+      <UIOutOfStockTitle>{title}</UIOutOfStockTitle>
+      <UIOutOfStockMessage>
+        {notificationMsgIcon} {notificationMsg}
+      </UIOutOfStockMessage>
+      <div>
```

Comment:
I think we can remove this extra `<div>.` It seems not doing much. Can you check it pls :)